pom.xml

```diff
@@ -184,19 +175,6 @@
     </pluginManagement>
 
     <plugins>
-      <plugin>
```

Comment:
unrelated nit: we don't use java <8 so we don't need special config related to retrolambda